data_steward/notebooks/bq.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not sure this file is needed anymore.

Reply:
There is one instance of bq in analytics (not part of the pipeline) that was not updated to use the utils module. I thought of creating a separate PR for it since it is not part of our pipeline and can cause linting errors which we are not too concerned about.